v3/src/components/axis/models/axis-model.ts

```diff
@@ -42,8 +42,8 @@ export const EmptyAxisModel = AxisModel
     min: 0,
     max: 0
   })
-export interface IEmptyAxisModel extends Instance<typeof CategoricalAxisModel> {}
-export interface IEmptyAxisModelSnapshot extends SnapshotIn<typeof CategoricalAxisModel> {}
+export interface IEmptyAxisModel extends Instance<typeof EmptyAxisModel> {}
+export interface IEmptyAxisModelSnapshot extends SnapshotIn<typeof EmptyAxisModel> {}
```

Comment:
This just fixes an old copy-paste error I noticed while using `EmptyAxisModel` as a reference. It doesn't have anything to do with the rest of the PR.